source/administration/mmctl-cli-tool.rst

```diff
@@ -21,11 +21,12 @@ This feature was developed to a large extent by community contributions and we'd
    - `mmctl channel`_ - Channel Management
```

Comment:
This doc page appears "broken" when clicking View file to get the preview.

Reply:
@amyblais I built the documentation locally without issues to verify.